### ISSUE TITLE
Landmark error with 'title-page' property 

### ISSUE THREAD

**Issue by atroia** [IDPF/epubcheck#564]
When using the 'title-page' landmark property in EPUB3, both 3.0.1 and 4.0.0a11 throw an error stating that 'title-page' is an undefined property, but if you take the hyphen out and use 'titlepage' it passes cleanly through both. The problem lies in that it is a valid property with the hyphen, and shouldn't be valid without it, as per the EPUB3 spec regarding Landmarks.

**Comment by mattgarrish** [comment]
It's an error, but not with epubcheck. Hyphenation was removed when the vocabulary was adapted from the original DAISY source, but it appears the identifiers in the n3 file that is used to generate the page didn't get changed so the old names are coming out in the underlying RDFa. In other words, although you can find this definition:

http://www.idpf.org/epub/vocab/structure/#title-page

The label "titlepage" you see on the page is the accepted property for use in EPUBs. Looks like the same was done with halftitle-page.

I've opened an issue in the EPUB tracker at [1] to get the vocab updated.

[1] https://github.com/IDPF/epub-revision/issues/577

**Comment by rdeltour** [comment]
Good catch! I'm closing the issue since there's nothing to be done in EpubCheck.

**Comment by atroia** [comment]
Wow, I haven't see that definition page before, thanks for posting it. Thank you for opening the issue to get the vocab updated, I was confused as to why everywhere I was looking had the hyphen and it was erroring out, but now it makes sense.

**Comment by mattgarrish** [comment]
I was assuming maybe it was getting harvested from the underlying IDs, but I also found it listed on the accessibility guidelines. That one was my mistake, as I was still more familiar with using the original DAISY names when I wrote that site; didn't spot the ID problem until you raised this issue. (I've fixed that page, too.)

It's possible that others picked up from one or the other mistake and propagated it, but it's definitely not correct.

Thanks for pointing it out!
